src/processes/follow-topic-process.js

```diff
@@ -121,8 +191,10 @@ const followTopicProcess = async (job, done) => {
         done(null, 'user is not following the topic');
         return;
       }
+
       // check if user is anonymous
-      const senderUser = await User.findByPk(communityMessageFormat.user_id);
+      const [senderUser] = await Promise.all([User.findByPk(communityMessageFormat.user_id)]);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `Promise.all` here seems unnecessary as there's only one promise being resolved. You could directly await the `User.findByPk` function.
```diff
- const [senderUser] = await Promise.all([User.findByPk(communityMessageFormat.user_id)]);
+ const senderUser = await User.findByPk(communityMessageFormat.user_id);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->